### PR TITLE
Upgrade apiextensions

### PR DESCRIPTION
**Pull request by ahadas**
Not for 2.4

### CODE REVIEW

File: tests/suit/framework/framework.go [hunk 1]
```diff
@@ -204,7 +204,7 @@ func (f *Framework) CreateNamespace(prefix string, labels map[string]string) (*v
 
 // AddNamespaceToDelete provides a wrapper around the go append function
 func (f *Framework) AddNamespaceToDelete(ns *v1.Namespace) {
-	f.namespacesToDelete = append(f.namespacesToDelete, ns)
+	//f.namespacesToDelete = append(f.namespacesToDelete, ns)
```
Comment (mmartinv) [inline tests/suit/framework/framework.go:207]: Is this change needed to upgrade API extensions?

**Comment by ahadas** [comment]
drop in favor of #407

**Comment by ahadas** [comment]
close in favor of #407
